### ISSUE TITLE
Home team table header shown twice

### ISSUE THREAD

**Issue by istvannagy** [EmbeddedTestingWorkshop/FixtureFinderNTD#110]
Steps to reproduce:
1. Go to the homepage of the application
2. Selected all countries to have search results
3. Check table headers 

Excepted result:
Home team header is shown only once and there is also a "guest team" header

Actual result:
Home team header is shown twice

![Uploading home team table header twice.PNG…]()

**Comment by istvannagy** [comment]
Duplicate of #117
